io.qt.qdbusviewer.yml

```diff
@@ -0,0 +1,29 @@
+app-id: io.qt.qdbusviewer
+runtime: org.kde.Sdk
```

Comment:
Is it on purpose that it uses Sdk as the runtime?

Reply:
Ok, I just realized the app actually comes from the Sdk. It would be better to figure out how to build it from source, so it can be updated separately from Sdk.